pkgs/development/php-packages/composer/default.nix

```diff
@@ -15,13 +15,14 @@
 
 stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "composer";
-  version = "2.7.9";
+  version = "2.8.1";
 
   # Hash used by ../../../build-support/php/pkgs/composer-phar.nix to
   # use together with the version from this package to keep the
   # bootstrap phar file up-to-date together with the end user composer
   # package.
-  passthru.pharHash = "sha256-tt5eZcGZ2AuhGJf74TZOBj6FjUg/aoGhdsTWDysdY0c=";
+  # passthru.pharHash = "sha256-5UcbEx1d5jEz73mTFTacifl6ykxm6yQw3wvkJQtINHs=";
```

Comment:
Please remove this extra comment line, it's not needed and confusing.

Reply:
@drupol Oops, I missed that, thanks.  Fixed.